### DISCUSSION TITLE
Custom domain (Cloudflare)

### DISCUSSION THREAD

**Discussion by L-Andy** [appwrite/appwrite#3764] [Q&A]
Hello appwriters.
I have been trying to get an SSL cert for my appwrite instance running on a linode server, by adding a custom domain to it. The domain is a cloudflare domain. Any solution wiill be of great use. because i don't seem to find any solution or hints on the internet and cloudflare is not part of manuals added in the appwrite docs

**Comment by stnguyen90** [comment]
@L-Andy, you may run into some issues using Cloudflare due to the proxy. I would suggest disabling the proxy (at least for the certificate generation part) and switching the SSL encryption mode to Flexible.

**Reply by stnguyen90** [reply]
@L-Andy, are you still having issues with Cloudflare?

**Comment by rumbis** [comment]
I would suggest  to disable even the flexible SSL in many cases  have redirect loops issues  also you got proxy IP from flexible SSL and some bot visits from the proxy Ip  that may mess or  get  brute forced by AI

**Comment by binaryfire** [comment]
We've run into a lot of issues with this too. Just opened a feature request which may help: https://github.com/appwrite/appwrite/issues/5010
